specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-02-28-preview/timeseriesinsights.json

```diff
@@ -1250,10 +1250,6 @@
             "name": "Kind",
             "modelAsString": false
           }
-        },
-        "properties": {
-          "x-ms-client-flatten": true,
-          "$ref": "#/definitions/EventSourceCommonProperties"
```

Comment:
You're removing the `properties` property which is a required property here. Semantic validation error.

Reply:
Fixed